definition-map.js

```diff
@@ -140,6 +140,26 @@ const QUERIES = {
             isPreview: "Requests the preview URL to launch SCORM content. This requires the current user to be in admin or manager role.",
             type: "The type of the Topic Or Course."
         }
+    },
+    CourseById: {
+        metadescription: "Returns various information about the course and the objects that are contained inside of the course, such as the sections, lessons, and pages associated with the course.",
+        args: {
+            id: "The ID of the course. Passing the ID of the CourseGroup will not work here. You can get the course ID from the Catalog Content query with the displayCourse field."
```

Comment:
`You can get the course ID from the Catalog Content query with the displayCourse field`

Personally think that part feels a little superfluous? You can get a Course ID from any queries that return a `Course` objectType. For example, you could get it from the `purchasedCourses` property in the `CurrentUser` query, or - with enough nesting - could also get it from the `UserRecentContent`, `UserContentItems`, and `UserWaitlist` queries, to name a few.

cc: @jackanticoti

Reply:
Perhaps we can change to you can get `displayCourse` under the `Course` object @rjschill87

Reply:
@jackanticoti The `displayCourse` property doesn't exist on the `Course` object though. If it really needs specifying, would maybe say the ID could be found on the `id` property of any query where a `Course` objectType can be returned.

Still though as far as documentation goes, it really feels like anything beyond "The ID of the course." is superfluous. From a dev perspective, if I were calling a query called `CourseById` and it takes an `id` arg, I would assume it's the ID of the course I'm querying for.

From a quick glance, it seems like Shopify's docs take the same approach: https://shopify.dev/api/storefront/2022-10/queries/collection

Reply:
Changed to "The ID of the course."